File: C# Fundamentals May 2019/04.Data Types and Variables/Java/src/RefactorVolumePyramid.java
import java.util.Scanner;
public class RefactorVolumePyramid {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Length: ");
        double length = Double.parseDouble(scanner.nextLine());

        System.out.print("Width: ");
        double width = Double.parseDouble(scanner.nextLine());

        System.out.print("Height: ");
        double heigth = Double.parseDouble(scanner.nextLine());

        heigth = (length * width * heigth) / 3;
        System.out.printf("Pyramid Volume: %.2f%n", heigth);

    }
}

File: C# Fundamentals May 2019/04.Data Types and Variables/PHP/RefactorVolumePyramid.php
<?php
$length = 0;
$width = 0;
$height = 0;

echo "Length: ";
$length = floatval(readline());
echo "Width: ";
$width = floatval(readline());
echo "Height: ";
$height = floatval(readline());
$result = ($length * $width * $height) / 3;
echo sprintf("Pyramid Volume: %.2f", $result) . PHP_EOL;


?>
